### PR TITLE
QoL Changes

### PR DESCRIPTION
**Pull request by DTraitor**
## About The Pull Request
Adds a few "Jump to X" verbs, makes "Open Marker UI" a verb instead of an action button (gotta make all those verbs UI buttons later). Fixed multi z movement for marker signals. Various necroqueue fixes 

## Why It's Good For The Game
Marker signal gameplay should be better now

## Changelog

:cl:
add: Added Jump to X verbs to marker signals
add: You can join/leave necroqueue at will now
fix: Necroqueue won't put players who left the game into necromorphs
fix: Marker signals can freely move between Z levels once again
/:cl:

### CODE REVIEW

File: deadspace/code/marker/markernet/marker_eye.dm [hunk 1]
```diff
@@ -185,6 +185,42 @@ GLOBAL_LIST_EMPTY(markers_signals)
...
+
+	if(!marker.active)
+		to_chat(src, span_notice("Marker is not active yet!"))
+		return
+	if(src in marker.necroqueue)
+		to_chat(src, span_notice("You are now in the necroqueue. When a necromorph vessel is available, you will be automatically placed in control of it. You can still manually posess necromorphs."))
+		marker.necroqueue += src
+	else
+		to_chat(src, span_notice("You have left the necroqueue."))
+		marker.necroqueue -= src
```
Comment (Kapu1178) [inline deadspace/code/marker/markernet/marker_eye.dm:200]: this smells wrong.

File: deadspace/code/marker/markernet/marker_eye.dm [hunk 2]
```diff
@@ -185,6 +185,42 @@ GLOBAL_LIST_EMPTY(markers_signals)
...
+
+/mob/camera/marker_signal/verb/jump_to_maker()
+	set name = "Jump to Marker"
+	set category = "Necromorph"
+
+	forceMove(get_turf(marker))
```
Comment (Kapu1178) [inline deadspace/code/marker/markernet/marker_eye.dm:206]: might want to use `abstract_move()`

File: deadspace/code/marker/markernet/marker_eye.dm [hunk 3]
```diff
@@ -185,6 +185,42 @@ GLOBAL_LIST_EMPTY(markers_signals)
...
+
+/mob/camera/marker_signal/verb/jump_to_necro()
+	set name = "Jump to Necromorph"
+	set category = "Necromorph"
+
+	var/mob/living/carbon/human/necromorph/necro = tgui_input_list(src, "Select necromorph to jump to", "Jump To Necromorph", marker.necromorphs)
+	if(necro)
+		forceMove(get_turf(necro))
```
Comment (Kapu1178) [inline deadspace/code/marker/markernet/marker_eye.dm:214]: see above

File: deadspace/code/marker/markernet/marker_eye.dm [hunk 4]
```diff
@@ -185,6 +185,42 @@ GLOBAL_LIST_EMPTY(markers_signals)
...
+
+/mob/camera/marker_signal/verb/jump_to_span_locs()
+	set name = "Jump to Necromorph Spawn Locs"
+	set category = "Necromorph"
+
+	var/atom/location = tgui_input_list(src, "Select object to jump to", "Jump To Spawn Loc", marker.necro_spawn_atoms)
+	if(location)
+		forceMove(get_turf(location))
```
Comment (Kapu1178) [inline deadspace/code/marker/markernet/marker_eye.dm:222]: see above

File: deadspace/code/marker/markernet/marker_eye.dm (hunk 2 above)
Comment (DTraitor) [reply deadspace/code/marker/markernet/marker_eye.dm:206]: `forceMove()` uses `abstract_move()` already (doesn't break anything so far)
```
/mob/camera/marker_signal/forceMove(atom/destination)
	abstract_move(destination) // move like the wind
	return TRUE
```

**Pull request merged**